src/packages/recompose/package.json

```
@@ -20,6 +20,6 @@
     "symbol-observable": "^4.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0 || ^16.0.0 || ^17.0.0"
+    "react": "^16.2.0 || ^17.0.0"
```

```suggestion
    "react": "^16.3.0 || ^17.0.0"
```

move out of this PR?